app/backend/main.py

```
@@ -65,8 +66,23 @@ async def lifespan(app: FastAPI):
     setup_langsmith()
 
+    # 주기적 인메모리 상태 정리 (60초마다)
+    async def _periodic_cleanup():
+        from app.backend.api.v1.endpoints.generation import _cleanup_stale_states
+
+        while True:
+            await asyncio.sleep(60)
+            try:
+                _cleanup_stale_states()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)